### PR TITLE
AssortedFixes(SeeDescription)

### PR DESCRIPTION
**Pull request by BloodCommando**
-dathomir_1_1.tab contains a moved vent that was reported in Issue #81
-item_stats.tab contains corrected EoL values for the ships added in GCW2
-mtp_corellia_times_vendor.tab previously had a sorosuub yacht, why?
-nova_schematic_vendor.tab previously had a sorosuub yacht, why?
-orion_schematic_vendor.tab previously had a sorosuub yacht, why?
-heroic_drops.tab contained the AT-RT in the loot table for the ISD instance, why?
-combat_data.tab fixes the Pulse Cannon's fire, which previously had a 20m radius, now corrected to 5m, as per all heavy weapons. This was a legacy bug from live.
-ep3_rryatt_krepauk_defeat_katarn.tab contained a barc speeder from a quest reward. This did not exist on live and I have removed it.
-shared_crashed_ship.tpf corrected a missing string found in a static object involved in Nym's Themepark.

### CODE REVIEW

File: sku.0/sys.server/compiled/game/datatables/item/vendor/mtp_corellia_times_vendor.tab [hunk 1]
```diff
@@ -9,4 +9,3 @@ item_meatlump_buff_device_map_01		2
 item_meatlump_buff_device_weapon_01		2			
 item_meatlump_buff_device_food_01		2			
 item_meatlump_buff_device_bomb_01		2			
-sorosuub	1000000
```
Comment (Cekis) [inline sku.0/sys.server/compiled/game/datatables/item/vendor/mtp_corellia_times_vendor.tab:12]: Leftover from Stella Bellum planned implementation for the Sorosuub

File: sku.0/sys.server/compiled/game/datatables/item/vendor/nova_schematic_vendor.tab [hunk 2]
```diff
@@ -1,5 +1,4 @@
 item	cash	resourceAmount	class	script	note
 s[none]	i[0]	i[1000]	i[0]	s	c
 item_reward_nova_engine_schematic_01_01		550			
-item_reward_nova_wpn_schematic_01_01		550
-sorosuub	1000000
```
Comment (Cekis) [inline sku.0/sys.server/compiled/game/datatables/item/vendor/nova_schematic_vendor.tab:5]: Leftover from Stella Bellum planned implementation for the Sorosuub

File: sku.0/sys.server/compiled/game/datatables/item/vendor/orion_schematic_vendor.tab [hunk 3]
```diff
@@ -2,4 +2,3 @@ item	cash	resourceAmount	class	script	note
 s[none]	i[0]	i[1000]	i[0]	s	c
 item_reward_orion_engine_schematic_01_01		550			
 item_reward_orion_wpn_schematic_01_01		550
-sorosuub	1000000
```
Comment (Cekis) [inline sku.0/sys.server/compiled/game/datatables/item/vendor/orion_schematic_vendor.tab:5]: Leftover from Stella Bellum planned implementation for the Sorosuub

File: sku.0/sys.server/compiled/game/datatables/loot/loot_items/dungeon/heroic_drops.tab [hunk 4]
```diff
@@ -19,9 +19,9 @@ item_heroic_random_ring_01_01		item_heroic_axkva_min_claw_light	item_heroic_dest
...
 item_heroic_random_ring_01_02		item_heroic_axkva_min_crystal_s02	item_heroic_destroyer_solar_console	item_heroic_destroyer_cooling_coil	item_heroic_destroyer_cooling_coil	item_heroic_destroyer_cooling_coil	st_pr_exar_kun_column_broken	st_pr_exar_kun_torch_02	st_pr_exar_kun_torture_table	item_heroic_exar_claw_s02	item_heroic_exar_claw_s02								
 item_limited_use_schematic_commando_ice_gun_04_01		item_heroic_axkva_min_crystal_s03	item_heroic_destroyer_space_beacon	item_heroic_destroyer_power_transformer	item_heroic_destroyer_power_transformer	item_heroic_destroyer_power_transformer	st_pr_exar_kun_column_broken	st_pr_exar_kun_torch_02	st_pr_exar_kun_zapper	item_heroic_exar_roots_s01	item_heroic_exar_roots_s01								
-item_heroic_tusken_meat_rack			item_deed_walker_06_02	item_heroic_destroyer_solar_console	item_heroic_destroyer_solar_console	item_heroic_destroyer_solar_console	item_limited_use_schematic_commando_ice_gun_04_01	item_limited_use_schematic_commando_ice_gun_04_01	st_pr_exar_kun_zapper	item_heroic_exar_roots_s02	item_heroic_exar_roots_s02
```
Comment (Cekis) [inline sku.0/sys.server/compiled/game/datatables/loot/loot_items/dungeon/heroic_drops.tab:22]: Leftover from Stella Bellum planned implementation for the AT-RT

**Comment by BloodCommando** [comment]
> Approving most of this... only question mark for me is issue #81 resolution. Bug says they're not clickable, yet moving them fixes the issue?

Yes, the original bug was that the object was placed in an area that the player could not reach. I lowered it to a position that can be easily clicked, while still preserving the visual integrity as much as possible. This has been the solution used by most other projects who have fixed this as far as I know.

**Comment by Cekis** [comment]
Nice... ok, then I'm good to go.  Good bunch of fixes!  :)

**Pull request merged**